### PR TITLE
Added missing implementations for Secure Storage feature

### PR DESCRIPTION
**Pull request by rleojoseph**
# Description
* Added Public interfaces for the Host app to clear storage and clear storage using MiniApp ID
* Refactored MiniAppSecureStorage not to have any Public interfaces.
* Added sample app implementation for Clearing Storages
* MINI-5015 fix

## Links
[MINI-4968](https://jira.rakuten-it.com/jira/browse/MINI-4968)
[MINI-5015](https://jira.rakuten-it.com/jira/browse/MINI-5015)

# Checklist
- [x] I have read the [contributing guidelines](CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

File: Sources/Classes/core/Display/RealMiniAppView.swift [hunk 1]
```diff
@@ -317,6 +317,14 @@ internal class RealMiniAppView: UIView {
     internal func presentAlert(alertController: UIAlertController) {
         UIApplication.topViewController()?.present(alertController, animated: true, completion: nil)
     }
+
+    func wipeAllSecureStorageForMiniApps() {
+        MiniAppSecureStorage.wipeSecureStorages()
+    }
+
+    func clearSecureStorage(for miniAppId: String) {
+        MiniAppSecureStorage.wipeSecureStorages()
```
Comment (laubengaier) [inline Sources/Classes/core/Display/RealMiniAppView.swift:326]: I think this should call `MiniAppSecureStorage.wipeSecureStorage(for miniAppId: String)`
```suggestion
        MiniAppSecureStorage.wipeSecureStorage(for: miniAppId)
```

**Review by laubengaier — APPROVED** [review]
one comment otherwise lgtm!

File: Sources/Classes/core/Storage/MiniAppSecureStorage.swift [hunk 2]
```diff
@@ -190,30 +190,73 @@ public class MiniAppSecureStorage: MiniAppSecureStorageDelegate {
...
+                }
+            } catch let error {
+                MiniAppLogger.d("🔑 Secure Storage Wipe Failed: \(name)", error.localizedDescription)
+            }
+        }
+    }
+
+    // MARK: - Clear storage for MiniApp ID
+    internal static func wipeSecureStorage(for miniAppId: String) {
+        if !miniAppId.isEmpty {
```
Comment (laubengaier) [inline Sources/Classes/core/Storage/MiniAppSecureStorage.swift:218]: just a question regarding this, the secure storage file will always be located in the miniapp directory so all versions can access it + the filename doesn't change so I don't think it's necessary to traverse or am I missing sth?

File: Sources/Classes/core/Storage/MiniAppSecureStorage.swift [hunk 3]
```diff
@@ -190,30 +190,73 @@ public class MiniAppSecureStorage: MiniAppSecureStorageDelegate {
...
-                MiniAppLogger.d("🔑 Secure Storage: ignored \(name)")
             }
         }
     }
 
-    public func clearSecureStorage() throws {
+    func clearSecureStorage() throws {
         MiniAppLogger.d("🔑 Secure Storage: destroy")
-        try FileManager.default.removeItem(at: MiniAppSecureStorage.storagePath(appId: appId))
+        self.storage?.removeAll()
```
Comment (laubengaier) [inline Sources/Classes/core/Storage/MiniAppSecureStorage.swift:245]: 👍

File: Sources/Classes/core/Storage/MiniAppSecureStorage.swift (hunk 2 above)
Comment (rleojoseph) [reply Sources/Classes/core/Storage/MiniAppSecureStorage.swift:218]: You are right man. Thanks! Updated them now

**Review by corycaywood — APPROVED** [review]

**Pull request merged**